Makefile

```diff
@@ -4,6 +4,9 @@ GO_FILES := $(shell find . -name '*.go' | grep -v /vendor/)
 VERSION := $(shell git describe --tags --always --long --dirty)
 TAG := $(shell git describe --abbrev=0 --tags)
 LDFLAGS = -ldflags "-X ${PKG}/cmd.wrVersion=${VERSION}"
+export GOPATH := $(shell go env GOPATH)
+PATH := $(PATH):${GOPATH}/bin
+SHELL := env PATH=${PATH} $(SHELL)
```

Comment:
What does this line do?

Reply:
The `go env` gets the default.
```
loan106830:~ dj3$ echo $GOPATH

loan106830:~ dj3$ go env GOPATH
/Users/dj3/go
```
`export` is required only for the glide install which requests `GOPATH` be set.

The `PATH` reset allows the freshly installed glide to be picked up if `${GOPATH}/bin` is not already on it - but for that to work we need to reset `SHELL` as well otherwise a cached `PATH` from when the Makefile started is used for all commands.

It does feel like I'm fighting `make` so would understand a not merge on taste grounds alone!

Reply:
This may become redundant once I move on from Glide, but for now a working solution is much appreciated!